content/docs/sidebar.json

```diff
@@ -92,7 +92,11 @@
         "slug": "fast-data-caching-hub"
       },
       "experiment-tracking",
-      "data-registries"
+      "data-registries",
+      {
+        "label": "ML Model Registries",
```

Comment:
"Model Registry" sounds better to me (ML is clear from the context), Registries is not needed -usually you would have a single registry per team

Reply:
Updated. Wait, should "Data Registries" also be singular?

Reply:
yep, I think it should be Data Registry

Reply:
Updated in https://github.com/iterative/dvc.org/pull/3447